lib/new_relic/agent/agent.rb

```diff
@@ -629,6 +635,10 @@ def create_and_run_event_loop
             @event_loop.on(interval_for SPAN_EVENT_DATA) do
               transmit_span_event_data
             end
+            @event_loop.on(interval_for CUSTOM_EVENT_DATA) do
+              # TODO: Change to log_event_data when present in collector response
```

Comment:
As noted in this `TODO` because we don't get a faster harvest response for the log endpoint yet, we're hijacking the one for custom events to get in on the 5-second harvest batching.